Erp multiple epochs

lawr158 opened the pull request.

lawr158 committed:
Update add_happe_v3_events_eeglab_struct.m

lawr158 committed:
Update update_file_proc_info_posthappe_v3.m

lawr158 committed:
Update batch_beapp_HAPPE_V3.m

lawr158 committed:
Update beapp2eeglab.m

lawr158 committed:
Update add_events_eeglab_struct.m

ybraverman commented on functions/add_happe_v3_events_eeglab_struct.m:
Looks good! Indexing into the current epoch of the file_proc_info.evt_info

ybraverman commented on functions/batch_beapp_HAPPE_V3.m:
Need to still check on line 61 edit (figuring out generalizing beapp2eeglab) other change looks good.

ybraverman committed:
Update update_file_proc_info_posthappe_v3.m

Added an if statement to only update tag information if segmentation in happe was run, got rid of the redundant | 1s

ybraverman commented on functions/update_file_proc_info_posthappe_v3.m:
Looks good - I added one if statement to only update event tags and number of segments pre/post rejection and then took out the redundant | 1s

ybraverman committed:
Update update_file_proc_info_posthappe_v3.m

ybraverman commented on functions/add_events_eeglab_struct.m:
```diff
@@ -48,10 +48,14 @@
 %     EEG.event(curr_event).behav_code=evt_info_curr_rec_period(curr_event).behav_code;
     
     % add event label, time latency, and sample number to EEGLAB structure
-    if isfield(evt_info_curr_rec_period,'type')
```
We need to keep this the same to generalize at other points in the pipeline, and will instead edit batch_beapp_HAPPE_v3.

Please remove these files/changes from the pull request!

ybraverman commented on functions/add_events_eeglab_struct.m:
Please remove this file from pull request (see comments in line)

ybraverman committed:
Update beapp2eeglab.m

Adding if statement for EEG.nbchan, file_proc_info.beapp_nchans_used is updated by epoch if prepp or ica have been run, otherwise there's just a value in the first position

ybraverman commented on functions/beapp2eeglab.m:
All set! I updated this script to be flexible depending on whether prep or ica were run or not - should pull for curr_rec_period if so and just from 1st position if not

ybraverman commented on functions/update_file_proc_info_posthappe_v3.m:
All set - made one change to flexibly access event updating only if happe_segment was turned on

ybraverman committed:
Update batch_beapp_HAPPE_V3.m

ybraverman commented on functions/batch_beapp_HAPPE_V3.m:
These changes look good. I also added a change to update file_proc_info so it gets updated appropriately in the eeglab events section

ybraverman committed:
Delete add_events_eeglab_struct.m
